jsonschema-core.xml

```diff
@@ -1471,51 +1460,41 @@
-                            encountered while evaluating an instance.
+                            The "$dynamicRef" keyword is an applicator that is used to reference a
+                            dynamically identified schema.
                         </t>
                         <t>
                             Together with "$dynamicAnchor", "$dynamicRef" implements a cooperative
                             extension mechanism that is primarily useful with recursive schemas
                             (schemas that reference themselves).  Both the extension point and the
-                            runtime-determined extension target are defined with "$dynamicAnchor",
-                            and only exhibit runtime dynamic behavior when referenced with
-                            "$dynamicRef".
+                            extension target are defined with "$dynamicAnchor", and only exhibit dynamic
+                            behavior when referenced with "$dynamicRef".
                         </t>
                         <t>
-                            The value of the "$dynamicRef" property MUST be a string which is
-                            a URI-Reference.  Resolved against the current URI base, it produces
-                            the URI used as the starting point for runtime resolution.  This initial
-                            resolution is safe to perform on schema load.
+                            The value of the "$dynamicRef" property MUST be a string which is a
+                            URI-Reference.  Resolved against the current URI base, it produces the URI used
+                            as the starting point for resolution.  This initial resolution is safe to perform
+                            on schema load.
                         </t>
                         <t>
                             If the initially resolved starting point URI includes a fragment that
                             was created by the "$dynamicAnchor" keyword, the initial URI MUST be
                             replaced by the URI (including the fragment) for the outermost schema
                             resource in the <xref target="scopes">dynamic scope</xref> that defines
                             an identically named fragment with "$dynamicAnchor".
-                            <cref>
-                                Requiring both the initial and final URI fragment to be defined
-                                by "$dynamicAnchor" ensures that the more common "$anchor"
-                                never unexpectedly changes the dynamic resolution process
-                                due to a naming conflict across resources.  Users of
-                                "$dynamicAnchor" are expected to be aware of the possibility
-                                of such name collisions, while users of "$anchor" are not.
-                            </cref>
```

Comment:
I removed this cref because it appears to be wrong. There's no way that an `$anchor` can change dynamic scope resolution and even if it could, it's not clear how bookending could possibly solve that problem. If anyone knows what this means, please share and I'll put it back in.